gym/envs/tests/test_envs.py

```diff
@@ -12,7 +12,12 @@ def should_skip_env_spec_for_tests(spec):
     # We skip tests for envs that require dependencies or are otherwise
     # troublesome to run frequently
 
-    # Skip mujoco tests for pull request CI
+    # Skipping if pull request
+    is_pr = os.environ.get('TRAVIS_PULL_REQUEST') != 'false' and os.environ.get('TRAVIS_PULL_REQUEST') is not None
```

Comment:
Why do we need to look for `TRAVIS_PULL_REQUEST`? The check below (looking for `MUJOCO_KEY_BUNDLE` or `~/.mujoco`) should subsume this.